assets/css/_variables.scss

```
@@ -1,6 +1,7 @@
 $font-size: (
   sm: .875rem,   // 14px
   md: 1rem,      // 16px
+  xxxl: 1.5rem,  // 24px
```

How about using `xl` instead?
```suggestion
  xl: 1.5rem,  // 24px
```